From 61668b8fcaa0f2a8a05100097284c0c427600033 Mon Sep 17 00:00:00 2001
From: George Joseph <george.joseph@fairview5.com>
Date: Mon, 2 May 2016 17:08:15 -0600
Subject: [PATCH] pjsip-apps/src/python/setup.py:  Take "make" from the
 environment

With "make" hard coded in setup.py, it chokes on FreeBSD because the system
make command isn't GNU compatibile.  This patch allows setup.py to take the
name of the make command from the MAKE environment variable if it exists.
If it doesn't, it defaults to "make".
---
 pjsip-apps/src/python/setup.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/pjsip-apps/src/python/setup.py b/pjsip-apps/src/python/setup.py
index 69a9859..ea1427d 100644
--- a/pjsip-apps/src/python/setup.py
+++ b/pjsip-apps/src/python/setup.py
@@ -60,25 +60,25 @@ if pj_version_suffix:
 	pj_version += "-" + pj_version_suffix
 
 #print 'PJ_VERSION = "'+ pj_version + '"'
-
+MAKE = os.environ.get('MAKE') or "make"
 
 # Fill in pj_inc_dirs
 pj_inc_dirs = []
-f = os.popen("make -f helper.mak inc_dir")
+f = os.popen("%s -f helper.mak inc_dir" % MAKE)
 for line in f:
     pj_inc_dirs.append(line.rstrip("\r\n"))
 f.close()
 
 # Fill in pj_lib_dirs
 pj_lib_dirs = []
-f = os.popen("make -f helper.mak lib_dir")
+f = os.popen("%s -f helper.mak lib_dir" % MAKE)
 for line in f:
     pj_lib_dirs.append(line.rstrip("\r\n"))
 f.close()
 
 # Fill in pj_libs
 pj_libs = []
-f = os.popen("make -f helper.mak libs")
+f = os.popen("%s -f helper.mak libs" % MAKE)
 for line in f:
     pj_libs.append(line.rstrip("\r\n"))
 f.close()
-- 
2.5.5

